### PR TITLE
Update for Maven based project build

### PR DESCRIPTION
**Pull request by merlintang**
Add the mvn supported built.

### CODE REVIEW

**Comment by merlintang** [comment]
@yuyongyang800  please review this and merge. thanks

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.github.merlintang</groupId>
```
Comment (yuyongyang800) [inline pom.xml]: Should we change this groupId to something more general?
Say "org.apache.spark.xxx".

**Review by yuyongyang800 — CHANGES_REQUESTED** [review]

**Review by yuyongyang800 — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (merlintang) [reply pom.xml]: ok, great, thanks, updated.

**Comment by merlintang** [comment]
@yuyongyang800  updated.

**Review by yuyongyang800 — APPROVED** [review]

**Pull request merged**